src/lib.rs

```diff
@@ -794,12 +798,12 @@ mod tests {
 
     #[test]
     fn test_lines() {
-        let in_buf: &[u8] = b"a\nb\nc";
+        let in_buf: &[u8] = b"This\nis\na sentence";
         let reader = RevBufReader::with_capacity(2, io::Cursor::new(in_buf));
         let mut it = reader.lines();
-        assert_eq!(it.next().unwrap().unwrap(), "c".to_string());
-        assert_eq!(it.next().unwrap().unwrap(), "b".to_string());
-        assert_eq!(it.next().unwrap().unwrap(), "a".to_string());
+        assert_eq!(it.next().unwrap().unwrap(), "a sentence".to_string());
+        assert_eq!(it.next().unwrap().unwrap(), "is".to_string());
+        assert_eq!(it.next().unwrap().unwrap(), "This".to_string());
         assert!(it.next().is_none());
     }
```

Comment:
I think it's ok to change the text to `This\nis\na sentence` for documentation purposes, but I'd leave it `a\nb\nc` in the test, so to keep the same pattern as the equivalent test in the standard lib:

https://github.com/rust-lang/rust/blob/8f3238f898163f09726c3d2b2cc9bafb09da26f3/library/std/src/io/buffered/tests.rs#L439

Reply:
Ah I see.

I thought it could be nice to have a test that ensures that the lines in the String are in the correct order and not reversed, but sticking to the standard lib implementation is definitely the right call :) 
I reversed this change.